### ISSUE TITLE
Empty labels still take up place in shiny 0.11

### ISSUE THREAD

**Issue by vzemlys** [rstudio/shiny#727]
Here is an example app:

```
library(ggvis)
library(dplyr)

ui <- shinyUI(
    fluidPage(
        titlePanel(
            h4("Test labels"), "Test labels"
            ),
        sidebarLayout(
            sidebarPanel(                
                conditionalPanel(
                    condition="input.condPanels==1",
		    selectInput("cxs_industry",label="Label",c("A"="A","B"="B"),selected="A",multiple=FALSE,width="100%"),	
		    selectInput("cxs_industry1",label="",c("A"="A","B"="B"),selected="A",multiple=FALSE,width="100%"),	
		    selectInput("cxs_industry2",label="Label",c("A"="A","B"="B"),selected="A",multiple=FALSE,width="100%"),	
		    selectInput("cxs_industry3",label="",c("A"="A","B"="B"),selected="A",multiple=FALSE,width="100%"),		
		    radioButtons("cx_trans",label="We have a label",choices=c("Actuals"=1,"Log"=4),selected=1,inline=TRUE),		   
		    radioButtons("cx_norm",label="",choices=c("Total"=1,"Per capita"=2),selected=1,inline=TRUE),
		    radioButtons("cx_norm1",label="",choices=c("Total"=1,"Per capita"=2),selected=1,inline=TRUE),
		    radioButtons("cx_norm2",label="We have a label",choices=c("Total"=1,"Per capita"=2),selected=1,inline=TRUE)			                 
                   ),
                conditionalPanel(condition="input.condPanels==4",
                                 downloadButton('c_download', 'Download data')
                                 )
                ),            
                mainPanel(
                tabsetPanel(                  
                    tabPanel("Compare countries",
			     ggvisOutput("devOrA"),
			      uiOutput("devOrA_ui"),
		                            value=1),
                    tabPanel("Data",                           
                             tableOutput("Data"),
                             value=4
                             ),
                    id="condPanels"
                    )
                )
            )
        )
    )	



server <- function(input, output, session) {

 devOrA <- reactive({
    data.frame(x=1,y=1)
  })
  devOrA %>%
    ggvis(x = ~x, y = ~1) %>%
    bind_shiny("devOrA","devOrA_ui", session)

}

shinyApp(ui, server)
```
Even if the labels are not present, the place for showing them is allocated. Makes spacing look weird, i.e. too much space around the elements.

Here is a screenshot with Safari web debugger, although there is no label, the space is still allocated:
![capture d ecran 2015-02-05 a 12 50 52](https://cloud.githubusercontent.com/assets/320871/6059022/c4355c86-ad35-11e4-8590-e13f3ae79509.png)

Maybe it can be fixed with some CSS magic, in that case I would be all ears.

**Comment by vzemlys** [comment]
I am running development version of shiny, as of commit b9c6b84ff5f120.

**Comment by wch** [comment]
You can make the space go away with `label=NULL`. I've updated the documentation with a bit more information about this.

**Comment by wch** [comment]
Also, from your example I noticed that the labels for inline radio buttons were too close to the radio buttons, so I've fixed that in 69aec85.

**Comment by vzemlys** [comment]
Thanks. It is kind of obvious, that `label=""` means an empty existing label, and `label=NULL` means no label. But somehow I missed it :)
